extended/src/main/java/io/kubernetes/client/extended/kubectl/Kubectl.java

```diff
@@ -308,6 +311,26 @@ protected GenericKubernetesApi<ApiType, KubernetesListObject> getGenericApi()
         throws KubectlException {
       return getGenericApi(apiTypeClass, KubernetesListObject.class);
     }
+
+    protected GenericKubernetesApi<? extends KubernetesObject, ? extends KubernetesListObject> getGenericApi(KubernetesObject targetObj)
+        throws KubectlException {
+      if (targetObj instanceof DynamicKubernetesObject) {
+        DynamicKubernetesObject obj = (DynamicKubernetesObject)targetObj;
+        String gv = obj.getApiVersion();
+        int ix = gv.indexOf("/");
+        String group = ix == -1 ? "" : gv.substring(0, ix);
+        String version = gv.substring(ix + 1);
```

Comment:
https://github.com/kubernetes-client/java/blob/master/util/src/main/java/io/kubernetes/client/util/ModelMapper.java#L180-L190

i think we can promote the above code from `ModelMapper` to a utility helper function like `ParsedAPIVersion ModelMapper#parseAPIVersion(...)`, so we can reuse that here, wdyt?

Reply:
Done.